bee-node/src/tools/password.rs

```diff
@@ -23,11 +25,17 @@ pub enum PasswordError {
 pub struct PasswordTool {}
 
 pub fn exec(_tool: &PasswordTool) -> Result<(), PasswordError> {
-    let password = read_password_from_tty(Some("Password: "))?;
-    let password_reenter = read_password_from_tty(Some("Re-enter password: "))?;
-    if password != password_reenter {
-        return Err(PasswordError::NonMatching);
-    }
+    let password = if let Ok(env_password) = env::var("BEE_TOOL_PASSWORD") {
+        env_password
+    } else {
+        let password = read_password_from_tty(Some("Password: "))?;
+        let password_reenter = read_password_from_tty(Some("Re-enter password: "))?;
+        if password != password_reenter {
```

Comment:
Can you compare to `read_password_from_tty` directly ? Since we don't actually use the result apart from the comparison

Reply:
Of course. I just thought it maybe would be more user readable this way :)